ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/StatementExecutorTest.java

```diff
@@ -97,14 +102,18 @@
   private KsqlEngine ksqlEngine;
   private StatementExecutor statementExecutor;
   private KsqlConfig ksqlConfig;
+  private HybridQueryIdGenerator hybridQueryIdGenerator;
 
   private final StatementParser mockParser = niceMock(StatementParser.class);
+  private final HybridQueryIdGenerator mockQueryIdGenerator =
```

Comment:
We're moving away from easy mock... fancy raising a PR to switch this test across to Mockito?  It's what I do when ever I need to touch an old easy mock test.

Reply:
I'll open one after this one gets merged.